### ISSUE TITLE
Additional vs VirtualFields?

### ISSUE THREAD

**Issue by paulspiteri** [keenondrums/types-redux-orm#4]
Can you possibly differentiate for me / in the docs, these two generic params?

thanks!

**Comment by fxlrnrpt** [comment]
@paulspiteri documentation on redux-orm is a mess, documentation on these typings carries on that great tradition of having many undocumented features and therefore it's no better. Sorry about that.
Basically, AdditionalFields were introduced as some kind of meta information. Usually it's something that doesn't correlate with 'core' business information about an entity (it's name, address, phone for user for instance), but something that you need for your GUI to fulfill your design (in my case it's fetch indicator). This division of fields into core fields and additional ones is required for typing purposes only. 
When I put put an entity that came from server in my DB I need only core fields, additional fields are set with defaults. When I read it with redux-orm it always has these additional fields. 
In other words 'write' requires input params to be of type Fields, yet read returns Fields & AdditionalFields.

As to VirtualFields, it's a feature of redux-orm that allows you to have regular virtual fields like with any other orm. For example, virtual field 'fullName', which is not persisted, but just a convenience wrapper for concatenating 'firstName' and 'lastName', which you have on you model.
